molecule/README.md

```diff
@@ -0,0 +1,93 @@
+# Introduction
+
+This directory is the home of the test playbooks:
+
+* ../tests/prepare.yml => setting up a VM or container with the minimum requirements (that usually is there, but not always in a container)
+* ../tests/playbook.yml => running a test against a VM or container (default postgres version: 13)
+
+# Molecule
+
+The default tested version is postgresql 9.6, 10, 11, 12, and 13 on Ubuntu 20.04. We are currently _not_ linting anything.
+
+```
+lint: |
+  set -e
+#  yamllint .
+#  ansible-lint
+```
+
+You can override this with setting the environment variable MOLECULE_DISTRO to one of:
+
+* centos7
+* centos8
+* fedora33
+* debian9
+* debian10
+* ubuntu1604
+* ubuntu1804
+* ubuntu2004
+
+The images we use are extended with systemd by Jeff Geerling. Before we start the tests, `molecule` runs the `prepare.yml` playbook to:
```

Comment:
I would provide here a link to JG repo with images Dockerfiles. Or, at least docker hub link?

Reply:
Before we start the tests, `molecule` runs the `prepare.yml` playbook to: -> Prior to the testing, `molecule` runs the `prepare.yml` playbook to